Obsolete MaxBodySizeToStore, moved to Audit instance (#4565)
Particular/docs.particular.net: servicecontrol/creating-config-file.md

In version 1.5.* and below, ServiceControl stores only the bodies of audit messages that are smaller than 100Kb.

> NOTE: This setting is obsolete from version 4 and higher. The setting is still supported in the Audit instance via setting [ServiceControl.Audit/MaxBodySizeToStore](/servicecontrol/audit-instances/creating-config-file.md#performance-tuning-servicecontrol-auditmaxbodysizetostore) if required

Reviewer: ```suggestion
NOTE: This setting is not available in versions 4.4 and higher. It is still supported in Audit instance via [ServiceControl.Audit/MaxBodySizeToStore](/servicecontrol/audit-instances/creating-config-file.md#performance-tuning-servicecontrol-auditmaxbodysizetostore) setting
```

> NOTE: This setting is obsolete from version 4.0.* and needs to be added to Audit instances via setting [ServiceControl.Audit/MaxBodySizeToStore](/servicecontrol/audit-instances/creating-config-file.md#performance-tuning-servicecontrol-auditmaxbodysizetostore).

Reviewer 2: ```suggestion
NOTE: This setting is obsolete from version 4 and higher. The setting is still supported in the Audit instance via setting [ServiceControl.Audit/MaxBodySizeToStore](/servicecontrol/audit-instances/creating-config-file.md#performance-tuning-servicecontrol-auditmaxbodysizetostore) if required
```
